### ISSUE TITLE
Unknown space

### ISSUE THREAD

**Issue by grupot3** [ethz-asl/voxblox#305]
Does voxblox give you also the unknown space (space that is behind an obstacle for example)?

Thanks!

**Comment by Magnusgaertner** [comment]
Yes it does, to every voxel there is a distance and a weight, the weight is what you are looking for.
